packages/neo4j-driver-lite/README.md

```diff
@@ -1,7 +1,10 @@
-# Neo4j Driver for JavaScript
+# Neo4j Driver for JavaScript (lite)
 
 This is the lite version of the official Neo4j driver for JavaScript.
 
+This version of driver has the same capabilities of the Neo4j Driver except for the support reactive session.
```

Comment:
Slight grammar correction:

This version of _the_ driver has the same capabilities _as_ the Neo4j Driver except for the support _of_ reactive _sessions_